### DISCUSSION TITLE
Catchall or auto create rules for default domain

### DISCUSSION THREAD

**Discussion by loeffelpan** [simple-login/app#2097] [Feature Requests]
I think a basic use-case for a selfhosted instance is to have a single (primary/default) domain without adding custom domains.
Unfortunately there is no way to setup "Catch all" or any "Auto Create"-Rule for the default domain.

Did I miss something?
Maybe it's worth a feature request (as an issue) to have the default domain visible under "Domains" to set those options for default domains?

**Comment by nguyenkims** [comment]
This indeed not currently supported in SL and a pull request would be appreciated!

**Reply by loeffelpan** [reply]
Thanks. As I'm not familiar with Python I categorized this discussion as a feature request, that someone else may develop.

**Comment by ukashazia** [comment]
Here's a temp workaround for this:
https://github.com/simple-login/app/issues/382#issuecomment-929994588
